app/services/l10n.js

```diff
@@ -14,20 +14,20 @@ export default class L10nService extends L10n {
   @computed(function() {
     return {
       'bn'      : 'বাংলা',
-      'zh_Hans' : '中文 (简化版)',
-      'zh_Hant' : '中文 (傳統的)',
+      'de'      : 'Deutsch',
       'en'      : 'English',
-      'fr'      : 'français',
-      'de'      : 'Deutsche',
-      'id'      : 'bahasa Indonesia',
-      'ko'      : '한국어',
-      'pl'      : 'Polskie',
       'es'      : 'Español',
-      'th'      : 'ไทย',
-      'vi'      : 'Tiếng Việt',
+      'fr'      : 'Français',
+      'ko'      : '한국어',
```

Comment:
Korean is last https://www.wikipedia.org/

Reply:
Sorry, I was referring to another wikipedia page, changed it now.